sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/CHANGELOG.md

```diff
@@ -1,15 +1,19 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
-
-### Features Added
+## 1.1.0 (2022-09-13)
```

Comment:
release 1.0.1

Reply:
Do you mean that all these 5 RPs' version should be 1.0.1？